api/google_api/routes/sheets.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const router = express.Router();
+const { SceGoogleApiHandler } = require('../util/SceGoogleApiHandler');
+const {
+  OK,
+  BAD_REQUEST
+} = require('../../util/constants').STATUS_CODES;
+
+router.post('/addToSpreadsheet', async (req, res) =>{
+  const scopes = ['https://sheets.google.com/'];
+  const pathToToken = __dirname + '/../../config/token.json';
+  const apiHandler = new SceGoogleApiHandler(scopes, pathToToken);
+  const sheetsID = '1gqIKkuNq_k9zMo9gUAdxhnp-N9X_5MO_6O3PoLPgZxM';
```

Comment:
Consider making this sheetsID an input for scalability. In case we want to add the functionality to doorcode onto a separate sheet.